src/dashboard/get_chart_data/templates/filter_numeric.sql.jinja

```diff
@@ -0,0 +1,48 @@
+{%- import 'syntax.sql.jinja' as syntax -%}
+
+{%- set ns =namespace(matches=[]) -%}
+{%- set filters=['eq', 'ne', 'gt', 'gte', 'lt', 'lte'] -%}
+
+{%- macro render_filter( data, filter_type, bound) -%}
+{%- if filter_type == 'eq'-%}
+{{ data }} = {{ bound }}
+{%- elif filter_type == 'ne'-%}
+{{ data }} != {{ bound }}
+{%- elif filter_type == 'gt'-%}
+{{ data }} > {{ bound }}
+{%- elif filter_type == 'gte'-%}
+{{ data }} >= {{ bound }}
+{%- elif filter_type == 'lt'-%}
+{{ data }} < {{ bound }}
+{%- elif filter_type == 'lte'-%}
+{{ data }} <= {{ bound }}
+{%- else -%}
+    not found {{ filter_type }}
+{%- endif -%}
+{%- endmacro -%}
+
+{%- macro is_filter_in_list(filter_configs) -%}
```

Comment:
OK so this is more about validating that the incoming filters are well-formed/valid rather than checking that they meet some kind of condition. Maybe `is_filter_valid` might be more immediately clear?